src/main/java/tictactoe/cli/ConsoleMediator.java

```diff
@@ -0,0 +1,76 @@
+package tictactoe.cli;
+
+import tictactoe.core.Board;
+import tictactoe.core.Mediator;
+import tictactoe.core.players.PlayerSymbol;
+
+public class ConsoleMediator extends Mediator {
+
+    private Console console;
+
+    public ConsoleMediator(Console console) {
+        this.console = console;
+    }
+
+    @Override
+    public void runGame() {
+        console.clear();
+        console.greetUser();
+        console.showGameOptions();
+        requestPlayersFromUI();
+    }
+
+    @Override
+    public void gameInstructions(Board board, PlayerSymbol playerSymbol) {
+        console.clear();
+        console.showBoard(board);
+        console.showMoveInstructions(board.getBoardSize(), playerSymbol);
+    }
+
+    @Override
+    public void requestMoveFromUI(Board board, PlayerSymbol playerSymbol) {
+        super.receiveMove(console.requestMove(board, playerSymbol));
+    }
+
+    @Override
+    public void moveSummary(int move, Board prevBoard, Board nextBoard, PlayerSymbol playerSymbol) {
+        console.clear();
+        console.showBoard(nextBoard);
+        console.showMoveSummary(move, prevBoard, playerSymbol);
+    }
+
+    public void playAgain() {
+        if (console.requestPlayAgain()) {
+            runGame();
+        } else {
+            console.goodbye();
+        }
+    }
+
+    @Override
+    public void requestBoardSizeFromUI() {
+        super.receiveBoardSize(console.requestBoardSize());
```

Comment:
Don't think you need reference to super?

Reply:
I thought it might make it clearer that it's referencing the abstract class' method but can remove it